extensions/binding/org.eclipse.smarthome.binding.ntp/src/main/java/org/eclipse/smarthome/binding/ntp/handler/NtpHandler.java

```diff
@@ -96,8 +96,8 @@ public void initialize() {
 
 			Configuration config = getThing().getConfiguration();
 			this.hostname = (String) config.get(PROPERTY_NTP_SERVER);
-			this.refreshInterval = (BigDecimal) config.get(PROPERTY_REFRESH_INTERVAL);
-			this.refreshNtp = (BigDecimal) config.get(PROPERTY_REFRESH_NTP);
+			this.refreshInterval = new BigDecimal (config.get(PROPERTY_REFRESH_INTERVAL).toString());

```

Comment:
It would be interesting to analyze the "how" then, because this sounds like a bug that you are now working around.